circuit/src/xdns/src/lib.rs

```diff
@@ -332,20 +333,68 @@ pub mod pallet {
         }
     }
 
-    // The build of genesis for the pallet.
+    /// The build of genesis for the pallet.
+    /// Populates storage with the known XDNS Records
     #[pallet::genesis_build]
     impl<T: Config> GenesisBuild<T> for GenesisConfig<T> {
-        fn build(&self) {}
+        fn build(&self) {
+            let circuit_dev: XdnsRecord<T::AccountId> = <XdnsRecord<T::AccountId>>::new(
+                b"wss://dev.net.t3rn.io".to_vec(),
```

Comment:
URLs ideally should reside in config files since they are subject to change.
Although I don't know how folks achieve that in substrate. 

At the minimum, we should attempt to take these URLs out and make them constant variables.
Same for `demo_gateway`

Reply:
I completely agree with this point. The solution would be to include a library providing environment variables for such URLs. To be implemented in an upcoming PR.